upgrade.md

```diff
@@ -75,6 +75,12 @@ When defining channel name placeholders in Laravel 5.3, the `*` character is use
         return (int) $user->id === (int) $userId;
     });
 
+### Commands
+
+#### Command Namespace Detector Trait Rename
+
+If any of your commands use the `AppNamespaceDetectorTrait` Trait, it has been renamed to `DetectsApplicationNamespace`.
```

Comment:
Best to include the full name. Also, no need for the "If any of your commands use" part. Just:

> The `...